Shared/UI/Views/ThemeLabel.swift

```diff
@@ -107,21 +107,19 @@ class ThemeLabel: UILabel {
         let lowercased = existingText.string.lowercased()
         
         guard let range = lowercased.range(of: text.lowercased()) else { return }
-                
+
         let attributedString = NSMutableAttributedString(existingText)
         attributedString.addAttributes(attributes, range: range.nsRange(text))
         super.attributedText = attributedString
     }
 
     private func setTextWithAttributes(_ newText: String) {
         let string = self.stringCasing.format(string: newText)
-        
-        let attributes = self.attributedText?.attributes(at: 0, effectiveRange: nil) ?? self.defaultAttributes
```

Comment:
This code would crash if the previous value was an empty string. There's really no need to access the previous text anyway, so we can always use the computed attributes.